sweep/walletsweep.go

```diff
@@ -30,6 +37,12 @@ func DetermineFeePerKw(feeEstimator lnwallet.FeeEstimator,
 	feePref FeePreference) (lnwallet.SatPerKWeight, error) {
 
 	switch {
+	// If both values are set, then we'll return an error as we require a
+	// strict directive.
+	case feePref.FeeRate != 0 && feePref.ConfTarget != 0:
+		return 0, fmt.Errorf("only FeeRate or ConfTarget should " +
+			"be set for FeePreferences")
+
```

Comment:
Should move to the commit where this func is introduced.

Reply:
I think it is best to do code move and alteration in different commits, to make it clear what behavior is actually changed :)